Hestia/app/src/androidTest/java/com/rugged/application/hestia/ActivatorAndActivatorStateTest.java

```diff
@@ -1,97 +0,0 @@
-package com.rugged.application.hestia;
-
-import android.support.test.runner.AndroidJUnit4;
-
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-import hestia.backend.Activator;
-import hestia.backend.ActivatorState;
-
-import static org.junit.Assert.*;
-
-@RunWith(AndroidJUnit4.class)
-public class ActivatorAndActivatorStateTest {
```

Comment:
Your coverage decreased by 22.59%, what went wrong here?
I expect the next iteration to fix this.